target/riscv/cpu_bits.h

```diff
@@ -506,6 +506,10 @@
 #define PTE_A               0x040 /* Accessed */
 #define PTE_D               0x080 /* Dirty */
 #define PTE_SOFT            0x300 /* Reserved for Software */
+#ifdef TARGET_CHERI
```

Comment:
Please guard by `!defined(TARGET_RISCV32)`, along with everything that uses these.

Reply:
Done